### PR TITLE
Enable audit log in local cluster

### LINKED ISSUE #42318: Enable audit log for hack/local-up-cluster.sh
**Is this a request for help?** (If yes, you should use our troubleshooting guide and community support channels, see http://kubernetes.io/docs/troubleshooting/.):

**What keywords did you search in Kubernetes issues before filing this one?** (If you have found any duplicates, you should instead reply there.):

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

**Kubernetes version** (use `kubectl version`):

**Environment**:
- **Cloud provider or hardware configuration**:
- **OS** (e.g. from /etc/os-release):
- **Kernel** (e.g. `uname -a`):
- **Install tools**:
- **Others**:

**What happened**:

**What you expected to happen**:
I want to set up a local cluster with audit log support

**How to reproduce it** (as minimally and precisely as possible):
```
./hack/local-up-cluster.sh
```

**Anything else we need to know**:
We can borrow sth from https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/container-linux/configure-helper.sh#L833-L846

### PR DESCRIPTION
**Pull request by xilabao**
fixes #42318

`ENABLE_APISERVER_BASIC_AUDIT=true hack/local-up-cluster.sh`

### CODE REVIEW

**Comment by k82cn** [comment]
@k8s-bot ok to test

**Comment by k82cn** [comment]
@xilabao , please update "ref to #42318" to "fixes #42318"; so the k8s-bot will close the issue when this PR merged.

**Comment by xilabao** [comment]
@k82cn Thanks for your reminding. updated

**Comment by xilabao** [comment]
@k8s-bot gci gke e2e test this
@k8s-bot cvm gke e2e test this

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -713,6 +733,7 @@ Local Kubernetes cluster is running. Press Ctrl-C to shut it down.
 
 Logs:
   ${APISERVER_LOG:-}
+  ${APISERVER_BASIC_AUDIT_LOG:-}
```
Comment (gyliu513) [inline hack/local-up-cluster.sh]: I think we should not printed this if not enabling the audit log.

**Comment by gyliu513** [comment]
>ENABLE_APISERVER_BASIC_AUDIT=true ENABLE_RBAC=true hack/local-up-cluster.sh

Why need enable RBAC for this?

File: hack/local-up-cluster.sh (hunk 1 above)
Comment (xilabao) [inline hack/local-up-cluster.sh]: fixed.

**Review by gyliu513 — COMMENTED** [review]
/lgtm thanks @xilabao

**Comment by xilabao** [comment]
@k8s-bot gci gke e2e test this
@k8s-bot cvm gke e2e test this
@k8s-bot cvm gce e2e test this

**Comment by xilabao** [comment]
@k8s-bot gci gke e2e test this
@k8s-bot cvm gke e2e test this
@k8s-bot cvm gce e2e test this

**Comment by xilabao** [comment]
@k8s-bot gci gke e2e test this

**Comment by xilabao** [comment]
@k8s-bot gci gke e2e test this

**Comment by xilabao** [comment]
@k8s-bot gci gke e2e test this

**Comment by k82cn** [comment]
@k8s-bot gce etcd3 e2e test this

**Comment by xilabao** [comment]
@k8s-bot kubemark e2e test this

**Comment by xilabao** [comment]
@k8s-bot unit test this

**Pull request merged**